skbio/io/tests/test_clustal.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
```

Comment:
nose doesn't like test files to be executable, so even having the shebang won't allows us to do either of these:

```
$ ./foo.py
$ foo.py
```